cmd/up.go

```diff
@@ -368,3 +378,20 @@ func printSvc(w io.Writer, c kubernetes.Interface, nss []string) error {
 	fmt.Fprintln(w)
 	return nil
 }
+
+func allReady(c kubernetes.Interface, nss []string) (bool, error) {
+	for _, ns := range nss {
+		pods, err := c.CoreV1().Pods(ns).List(metav1.ListOptions{
```

Comment:
I think this should be a List across all namespaces, rather than a for-loop across explicit namespaces.

Reply:
👍